content/en/developers/integrations/log_pipeline.md

```diff
@@ -130,14 +184,14 @@ To start a review process, export your log pipeline and relevant custom facets u
 
 {{< img src="developers/integrations/export_pipeline.png" alt="Click the Export Pipeline icon to export your log pipeline in Datadog" width="50%">}}
 
-Include sample raw logs with all the attributes you expect to be sent into Datadog by your integration. Raw logs comprise of the raw messages generated directly from the source before they have been ingested by Datadog.
+Include sample raw logs with **all** the attributes you expect to be sent into Datadog by your integration. Raw logs comprise of the raw messages generated directly from the source application **before** they are sent to Datadog.
 
 Exporting your log pipeline includes two YAML files:
 
-- One with the log pipeline, which includes custom facets, attribute remappers, and grok parsers.
-- One with the raw example logs with an empty result. 
+* One with the log pipeline, which includes custom facets, attribute remappers, and grok parsers. The exported file is named `pipeline-name.yaml`.
+* One with the raw example logs with an empty result. The exported file is named `pipeline-name_test.yaml`
```

Comment:
what does it mean "empty result"?

Reply:
It's part of the submission process. In a next version we can add images to describe this.